packages/web-components/fast-foundation/src/accordion/accordion.spec.md

```diff
@@ -73,6 +73,32 @@ As defined by the W3C:
 - start
 - end
 
+*Observable*
+- `expandedIcon` - A property used to override the default icon in the `expanded-icon` slot.
+- `collapsedIcon` - A property used to override the default icon in the `collapsed-icon` slot.
+
+```ts
+const expandedIcon = html`
+        <svg>
+            <path/>
+        </svg>
+`
+
+ const collapsedIcon = html`
+    <svg>
+        <path/>
+    </svg>
+ `
+
+export class MyAccordionItem extends AccordionItem {
```

Comment:
While an author could absolutely create a new element that extends AccordionItem, I think a more common way to utilizes these properties would be to override the element instance *in a template*. Doing so is easier because it doesn't require defining a new element just change the properties. I think we should show this example as well:

```ts
html`
    <fast-accordion-item :expandedIcon=${expandedIcon}></fast-accordion-item>
`
```